activation/activation.go

```diff
@@ -197,7 +197,11 @@ func (b *Builder) buildNipstChallenge(epoch types.EpochId) error {
 	}
 	seq := uint64(0)
 	prevAtxId := *types.EmptyAtxId
-	if b.prevATX != nil {
+	commitmentMerkleRoot := []byte(nil)
+
+	if b.prevATX == nil {
```

Comment:
you already check this condition a few lines above

Reply:
The previous section is about fetching `prevATX` if exists, and this one is about handling different stuff whenever it exist or not. 
I thought putting the new logic in the previous section would be confusing and easy to miss.